dimod/sampleset.py

```diff
@@ -1083,6 +1083,55 @@ def append_variables(self, samples_like, sort_labels=True):
                                        sort_labels=sort_labels,
                                        **self.data_vectors)
 
+    def lowest(self, rtol=1.e-5, atol=1.e-8):
+        """Return a SampleSet containing the samples with the lowest energy.
+
+        Args:
+            rtol (float, optional, default=1.e-5):
+                The relative tolerance. See :func:`numpy.isclose`.
```

Comment:
In this case it is more nuanced than that, which is why I refer the user elsewhere. It is a link in RTD.

Reply:
I agree that the link is helpful and agree that we shouldn't bring in all the nuance from a dependency, but I think we should provide more info here on what each parameter does. You can even state that they should look at the :func:'numpy.iclose` for details. Some users just need a reminder of what is what when they're looking at the code plus it's helpful for inline command help where they otherwise need to google for the reference.

Reply:
Take a look at the latest version